src/components/sidenav/sidenav.spec.js

```diff
@@ -52,18 +52,18 @@ describe('mdSidenav', function() {
       expect($rootScope.show).toBe(false);
     }));
 
-    it('should show no backdrop if disabled', inject(function($rootScope, $material, $timeout) {
-      var el = setup('md-disable-backdrop="true"');
+    it('should show a backdrop if md-disable-backdrop is set to false', inject(function($rootScope, $material) {
+      var el = setup('md-is-open="show" md-disable-backdrop="false"');
       $rootScope.$apply('show = true');
 
       $material.flushOutstandingAnimations();
 
       var backdrop = el.parent().find('md-backdrop');
-      expect(backdrop.length).toBe(0);
+      expect(backdrop.length).toBe(1);
     }));
 
-    it('should show no backdrop if disabled', inject(function($rootScope, $material, $timeout) {
-      var el = setup('md-disable-backdrop');
+    it('should not show a backdrop if md-disable-backdrop is set to true', inject(function($rootScope, $material) {
+      var el = setup('md-is-open="show" md-disable-backdrop="true"');
```

Comment:
Yes, `md-disable-backdrop` is also working. I just used a value in the test, to clarify the current use case.